react-web/src/components/JiraLoginForm.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Form, FormGroup, ControlLabel, FormControl, Button, HelpBlock } from 'patternfly-react';
+
+export class JiraLoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hostEntered: false,
```

Comment:
Are these values to check if input is pristine/dirty? Maybe we can create a component to handle this?
Jameel has something similar too =>https://github.com/maleck13/storypoint/blob/master/react-web/src/components/UsernameForm.js#L9